### DISCUSSION TITLE
Merge VK UI from 4 to 5

### DISCUSSION THREAD

**Discussion by Vlad-Remnev** [VKCOM/VKUI#5078] [Q&A] [answered]
Hi? I want to discuss one problem about the project, I'm making. Version of VK UI in my project is 4.28.1 and now I need to update to 5. But in one place of code, I have this:
'<div {...restProps} className={getClassName('HorizontalScroll', platform)}>'
The problem is getClassName, which is in fifth cersion is depricated, and i don't know how to change the writing here, using only usePlatfrom (in code it is platform).
Thank for future helping, if somebody will give the answer.

**Comment by inomdzhon** [comment, marked as answer]
Hi!

You could do something like this:

```tsx
import { type PlatformType, classNames, usePlatform } from "@vkontakte/vkui";

function getClassNameWithPlatform(platform: PlatformType) {
  switch (platform) {
    case 'android':
      return 'Example--android';
    case 'ios':
      return 'Example--ios';
    default:
      return;
  }
}

const Example = () => {
  const platform = usePlatform();

  return <div className={classNames('Example', getClassNameWithPlatform(platform)} />
};
```
